specs/eid-hermes-commands-format.md

```diff
@@ -0,0 +1,214 @@
+# eid-hermes Commands
+
+## Introduction
+
+This document defines the commands that can be sent to a Hermes device. It
+includes both the commands' format as well as which commands are available.
```

Comment:
It also defines the command completions which will be sent by the device back to the host.